changelog/exitcodes.dd

```diff
@@ -0,0 +1,11 @@
+DUB command exit codes have been adjusted
+
+Some certain dub command conditions now return exit code 2 instead of exit code 1. Exit code 1 is now always used for usage errors, while exit code 2 is the more generic any error occured or package failed to load.
+
+The following commands could now return exit code 2 instead of exit code 1 when failing to load packages or other errors occur:
+
+- dub clean
+- dub add
+- dub search
+- dub convert
+
```

Comment:
```suggestion
- dub clean
- dub add
- dub search
- dub convert
```

Reply:
![this suggestion is invalid because no changes were made to the code](https://wfr.moe/f6fOoH.png)

Reply:
Remove trailing line